xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-ui/src/main/resources/XWiki/Ratings.xml

```diff
@@ -394,9 +394,10 @@ position: relative;
     <className>XWiki.UIExtensionClass</className>
     <guid>d2420ff0-a5b9-4fae-a798-fc623aac7a6b</guid>
     <property>
-      <content>{{include reference='XWiki.RatingsMacros'/}}
-
-{{velocity}}
+      <content>{{velocity}}
+#if (!$isGuest)
```

Comment:
Hi, Vincent!

I was just saying that if the current user doesn't have view rights on anything related to the ratings logic (ex. XWiki.RatingsMacros) there's no point in evaluating the code in the UI extension because it's going to produce errors (like the one described in the jira issue).

As for restricting guest users voting capability, this is another issue which is handled [here.](https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-ui/src/main/resources/XWiki/RatingsMacros.xml#L59)
I thought this was just about displaying.